src/pages/OrderBook.tsx

```diff
@@ -59,12 +59,12 @@ const OrderBookWrapper = styled.div`
   }
 
   ${InputBox} {
-    padding 0 0 0 3rem;
+    padding: 0 0 0 3rem;
```

Comment:
shouldn't this have be shown as a compile error or something?

Reply:
Eslint doesn't catch this or probably anything for css-in-js
Anyone wants to setup [stylelint](https://styled-components.com/docs/tooling#stylelint) ?

Reply:
would just be an improper css rule and do nothing bad (error wise) because of the semi-colon